packages/Button/src/Button.tsx

```diff
@@ -397,7 +395,7 @@ export default class Button extends Component<ButtonProps> {
 
     if (usesLinkElement(this.props)) {
       return (
-        <a {...rest as HTMLElementProps<'a'>} {...commonProps}>
+        <a {...(rest as HTMLElementProps<'a'>)} {...commonProps}>
```

Comment:
Weird... Prettier _removed_ the parens for me when I had originally put them in.

Reply:
Prettier kept adding them for me -- couldn't take them out!